themes/socialbase/templates/node/node--teaser.html.twig

```diff
@@ -93,7 +93,7 @@
 
     {% if view_mode == 'teaser' or no_image %}
       {% block card_teaser_type %}
-        <a href="{{ url }}">
+        <a href="{{ url }}" title="{{- label|render|striptags|trim -}}">
```

Comment:
Title's don't help accessibility. It does help sighted users to figure out what the icon is for. 

I suggest the following:

- Before the SVG add the node bundle type as text that is only visible to screenreaders (Drupal has a class for this)
- Change the title to have this same text so that sighted users can see what the icon means when they hover over it

See: https://silktide.com/blog/2013/i-thought-title-text-improved-accessibility-i-was-wrong